public/config.json

```diff
@@ -23,6 +23,6 @@
     "sample_format_external_result_import": "/External-Results-Template.csv",
     "enable_abdm": true,
     "enable_hcx": false,
-    "enable_scribe": false,
+    "enable_scribe": true,
```

Comment:
```suggestion
    "enable_scribe": false,
```

Reply:
we are not enabling scribe in develop right . so i think its a accidental change ?

Reply:
Yes, accidental change sorry